rfcs/0000-universal-asset-notation/0000-universal-asset-notation.md

```diff
@@ -0,0 +1,118 @@
+
+## Definition
+
+The Universal Asset Notation (UAN) consists of asset notation and route notation, which in turn are build upon asset symbol, chain symbol and bridge symbol.
+
+UAN Definition (in [BNF](https://en.wikipedia.org/wiki/Backus%E2%80%93Naur_form)):
+
+```
+<UAN> ::= <asset> | <asset> "|" <route>
+<asset> ::= <asset-symbol> "." <chain-symbol>
+<route> ::= <path> | <path> "|" <route>
+<path> ::= <bridge-symbol> "." <chain-symbol>
+<asset-symbol> ::= <upper-char>
+<chain-symbol> ::= <lower-char>
+<bridge-symbol> ::= <lower-char>
+<upper-char> ::= <upper-case> | <digit>
+<lower-char> ::= <lower-case> | <digit>
+<upper-case> ::= "A" | "B" | "C" | "D" | "E" | "F" | "G" | "H" | "I" | "J" | "K" | "L" | "M" | "N" | "O" | "P" | "Q" | "R" | "S" | "T" | "U" | "V" | "W" | "X" | "Y" | "Z"
+<lower-case> ::= "a" | "b" | "c" | "d" | "e" | "f" | "g" | "h" | "i" | "j" | "k" | "l" | "m" | "n" | "o" | "p" | "q" | "r" | "s" | "t" | "u" | "v" | "w" | "x" | "y" | "z"
+<digit> ::= "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+```
+
+- Characters `|` and `.` are chosen in order to avoid confliction with `-` widely used in LP token in DeFi applications.
+- `<asset>` is the first component of UAN. It is a pair of `<asset-symbol>` and `<chain-symbol>` connected by a dot `.`, e.g. `BTC.ckb` for bitcoin resides on the CKB chain, `CKB.gw` for CKByte resides on the Godwoken chain.
+- `<route>` is the route through which the asset come to Nervos Network, it's the second component of UAN. `<route>` is only required when the asset's chain of residence is different from its chain of issuance. `<route>` may consist of one or many `<path>`. If it's a multi-paths route, all paths are connected by `|`.
+- `<path>` is a pair of bridge symbol and chain symbol, connected by a single dot `.`.
+- `<asset-symbol>` is a upper-case abbreviation of the asset, e.g. `BTC` for bitcoin, `CKB` for CKByte.
+- `<chain-symbol>` is a lower-case abbreviation of the source chain where the asset resided before came through the path, e.g. `eth` for the Ethereum chain, `ckb` for the CKB chain. Note the source chain may be different from the issuance chain of the asset.
+- `<bridge-symbol>` is a lower-case abbreviation of the bridge through which the asset came, e.g. `fb` for assets moved by Forcebridge.
+
+Examples:
+
+```
+// CKByte on the CKB chain
+CKB.ckb
+
+// CKByte on the Godwoken chain
+CKB.gw
+
+// WBTC crossed from the Ethereum chain to the CKB chain via Forcebridge
+WBTC.ckb|fb.eth
+
+// BNB crossed from the BSC chain to the CKB chain via Forcebridge
+BNB.ckb|fb.bsc
+
+// ETH crossed from the Ethereum chain to the CKB chain via Anotherbridge
+ETH.ckb|ab.eth
+
+// ETH crossed from the Ethereum chain to the Godwoken chain via Forcebridge and Godwoken bridge
```

Comment:
```suggestion
// ETH crossed from the Ethereum chain to the Godwoken chain via Force Bridge and Godwoken Bridge
```

Reply:
Follow the name style of Force Bridge

## References:
- https://github.com/nervosnetwork/force-bridge/pull/373#discussion_r924029488
- https://github.com/nervosnetwork/rfcs/pull/335/files#r913060381